astronomer/providers/dbt/cloud/sensors/dbt.py

```diff
@@ -0,0 +1,60 @@
+import time
+from typing import Any, Dict
+
+from airflow import AirflowException
+from airflow.providers.dbt.cloud.sensors.dbt import DbtCloudJobRunSensor
+from airflow.utils.context import Context
```

Comment:
See similar comment about importing this module for typing.

Reply:
Same here